.github/workflows/github-ci.yml

```diff
@@ -0,0 +1,23 @@
+name: Build
+on: [push, pull_request]
+jobs:
+  generator-jhipster-nav-element:
+    name: npm test
+    runs-on: ${{ matrix.os }}
+    if: "!contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]')"
+    timeout-minutes: 20
+    strategy:
+      fail-fast: false
+      matrix:
+        node_version: [12.14.0]
+        os: [ubuntu-latest]
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: Run git history
+        run: git --no-pager log -n 10 --graph --pretty='%Cred%h%Creset -%C(yellow)%d%Creset %s %Cgreen(%cr) %C(bold blue) <%an>%Creset' --abbrev-commit
+      - run: npm ci
+      - run: npm test
+    #  - run: npm run coveralls
```

Comment:
enable coveralls please - I think the issue with coveralls is fixed

Reply:
To add this without fail, maybe you have to include COVERALLS_REPO_TOKEN variable in this repo: lemurheavy/coveralls-public#632 .